index.html

```diff
@@ -4,9 +4,234 @@
+				event is expected to welcome nearly 100 exhibiting companies and 4,000 + attendees.
+			</p>
+			<h2 class="headline-date">
+					2023.02.23(THU) ~ 25(SAT)
+			</h2>
+			<p class="headline-paragraph-location">
+					@ Ethiopian Skylight Hotel
+			</p>
+		</div>
+	</div>
+</header>
+<main>
+	<div class="container">
+		<div class="main-title">
+			<p>Main Program</p>
+		</div>
+		<div class="main-program-container">
+			<div class="main-cards">
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="./img/icon/lecture-icon.png">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Lecture
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Listen to the speakers from various countries about the messeges of sharing and opening
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/exhibit-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Exhibition
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Appreciate various creations applying CC license of artists, organized from Art Center
+							Nabi.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/forum-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Forum
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Have the time to share your thoughts and opinions with experts for each topic.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/workshop-icon.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Workshop
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Try creating your own work using open source license rather than just watching at it.
+						</p>
+					</div>
+				</div>
+				<div class="main-card">
+					<div class="main-card-img">
+						<img src="img/icon/discussion.png" alt="">
+					</div>
+					<div class="main-card-title">
+						<p>
+							Discussion
+						</p>
+					</div>
+					<div class="main-card-description">
+						<p>
+							Get opportunities to network with CC affiliates around the world, also after the summit.
+						</p>
+					</div>
+				</div>
+			</div>
+		</div>
+		<button type="button" class="main-btn">
+			Join Ethio Health Exibition 2023
+		</button>
+		<a href="#" class="main-link-btn">See the all program</a>
+	</div>
+</main>
+<section class="container speakers">
+	<div class="speaker-title">
+		<p>Featured Speakers</p>
+	</div>
+	<div class="speaker-container-card">
+		<!-- here will be the speakers -->
+	</div>
+	<button type="button" class="speakers-btn">
+		MORE
+			<i class="fa fa-chevron-down" ></i>
+	</button>
```

Comment:
- [ ] Please make sure that the images here match the design add the chess-like image for each image you are free with the content but not the design.
![image](https://user-images.githubusercontent.com/74744787/218276923-a7da752c-eade-4f6e-bd66-c1184eec6707.png)

![image](https://user-images.githubusercontent.com/74744787/218276908-115c76dc-49fc-4370-abba-993ea7359aac.png)